### PR TITLE
Update rest-api 0.2.0

### PR DESCRIPTION
**Pull request by nivemaham**
Update rest-api 0.2.0

### CODE REVIEW

**Comment by yatharthranjan** [comment]
Can we also remove rest api from /images

File: dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml [hunk 1]
```diff
@@ -231,17 +231,19 @@ services:
...
     depends_on:
       - hotstorage
+      - managementportal-app
     restart: always
     volumes:
-      - "./etc/rest-api:/usr/local/tomcat/conf/radar"
+      - "./etc/rest-api:/usr/local/conf/radar/rest-api"
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8080/radar/api/swagger.json"]
+      test: ["CMD", "curl", "-f", "http://localhost:8080/api/openapi.json"]
```
Comment (yatharthranjan) [inline dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml]: Please use `wget --spider http://localhost:8080/api/openapi.json` instead of curl

**Comment by yatharthranjan** [comment]
@nivemaham status of this ?

**Comment by nivemaham** [comment]
Will have a look.

**Pull request merged**